src/components/FilterControls/MultiSelect/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react'
+import OutlinedInput from '@mui/material/OutlinedInput'
+import InputLabel from '@mui/material/InputLabel'
+import MenuItem from '@mui/material/MenuItem'
+import FormControl from '@mui/material/FormControl'
+import Select, { SelectChangeEvent } from '@mui/material/Select'
+import Checkbox from '@mui/material/Checkbox'
+import ListItemText from '@mui/material/ListItemText'
+
+interface MultiSelectProps {
+  options: string[]
+  defaultOptions: string[]
+  onClose: (selectedOptions: string[]) => void
+  label: string
+}
+
+const MultiSelectCheckmarks: React.FC<MultiSelectProps> = ({
+  options,
+  defaultOptions,
+  onClose,
+  label,
+}) => {
+  const [internalSelectedOptions, setInternalSelectedOptions] =
```

Comment:
Similar to my other comments, if we make this non-controlled, we should be able to remove internal state management here entirely and just. use the onClose callback.

On a side note, are we sure we only want to invoke changes on `onClose`? As opposed to `handleChange`?